src/Strategy/Python/Poetry.hs

```diff
@@ -137,6 +141,12 @@ analyze PoetryProject{pyProjectToml, poetryLock} = do
           , dependencyManifestFiles = [poetryLockPath lockPath]
           }
     Nothing -> do
+      _ <-
+        recover
```

Comment:
[nit]: use `void` instead.  Will require running the formatter.

```suggestion
      void . recover
```

Reply:
resolved in: https://github.com/fossas/fossa-cli/pull/803/commits/93cdf4ecf7fb004f155f7337c9c850dd29e1189d